policies/tagging/README.md

```diff
@@ -12,6 +12,9 @@ All environments and resources:
 ## Allowed values
 Tags and their allowed values.
 
+application:
+ - "core"
```

Comment:
core is not correct, this should link to the data dictionaries, https://github.com/hmcts/terraform-module-common-tags/blob/master/team-config.yml and https://github.com/hmcts/cnp-jenkins-config/blob/master/team-config.yml